docs/keyring.rst

```diff
@@ -38,6 +38,16 @@ You can fetch the username as well::
 
 Or really any kind of parameter in a storage section.
 
+It might be recommended to use an external password manager such as ``pass``
```

Comment:
"It might be recommended" sounds weird, I suggest something like "For example".

Also it might be better to introduce a new headline here.